.github/workflows/reused-commit-msgs-validator.yml

```diff
@@ -8,10 +8,12 @@ on:
 jobs:
   validate-commit-message:
     name: Validate Commit Message
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
What's the point of specifying the exact runner machine? This job just validates commit messages; it doesn't build anything, so we don't care about what runs it.